spark/src/main/scala/com/cloudera/livy/spark/client/ClientSession.scala

```diff
@@ -65,6 +71,14 @@ class ClientSession(val sessionId: Int, createRequest: CreateClientRequest)
     performOperation(job, false)
   }
 
+  def addFile(fileData: Array[Byte], fileName: String): Unit = {
```

Comment:
I thought this was the only way scalatra supported it, but it looks like we can actually write to a local file and then copy that over to HDFS. I will change this to do that.